plutus-pab/src/Plutus/PAB/Webserver/Handler.hs

```diff
@@ -206,13 +225,13 @@ walletProxy ::
     :<|> (WalletId -> Tx -> PABAction t env Tx))
 walletProxy createNewWallet =
```

Comment:
If we started the PAB with the `RemoteClient`, then we shouldn't call `walletProxy`.

Create a Jira task to remove this `WalletProxy` to work in the future.

Reply:
https://input-output.atlassian.net/browse/SCP-3096